client/lib/feed-post-store/index.js

```diff
@@ -268,7 +180,8 @@ function receiveError( feedId, postId, error ) {
 		}
 	}
 
-	setPost( postId, {
+	setPost( {
+		global_ID: uuid(),
```

Comment:
this line is icky. errors need to live in state, and we need to access it by postKey, but we don't have a real global_ID for them necessarily so I invent one here?